protocol-designer/src/components/modals/CreateFileWizard/EquipmentOption.tsx

```diff
@@ -102,6 +134,66 @@ export function EquipmentOption(props: EquipmentOptionProps): JSX.Element {
     )
   } else if (showCheckbox && disabled) {
     iconInfo = <Flex width="1.5rem" />
+  } else if (multiples != null) {
+    const { numItems, maxItems, isDisabled } = multiples
+    let upArrowCSS = ARROW_STYLE
+    if (isDisabled || numItems === maxItems) {
+      upArrowCSS = ARROW_STYLE_DISABLED
+    } else if (numItems > 0) {
+      upArrowCSS = ARROW_STYLE_ACTIVE
+    }
+    let downArrowCSS = ARROW_STYLE
+    if (numItems === 0) {
+      downArrowCSS = ARROW_STYLE_DISABLED
+    } else if (numItems > 0) {
+      downArrowCSS = ARROW_STYLE_ACTIVE
+    }
+
+    iconInfo = (
+      <Flex
+        flexDirection={DIRECTION_COLUMN}
+        gridGap={SPACING.spacing4}
+        width="1.5rem"
+        alignItems={ALIGN_CENTER}
+      >
+        <Flex
+          {...tempTargetProps}
+          data-testid="EquipmentOption_upArrow"
+          onClick={
+            numMultiples === 7
+              ? undefined
+              : () => {
+                  multiples.setValue(numMultiples + 1)
+                  setNum(numMultiples + 1)
+                }
+          }
+        >
+          <Icon css={upArrowCSS} size={SPACING.spacing12} name="ot-arrow-up" />
```

Comment:
let's avoid using SPACING for width, height, and size.

```suggestion
          <Icon css={upArrowCSS} size="0.75rem" name="ot-arrow-up" />
```